### DISCUSSION TITLE
Consistent handling of formats

### DISCUSSION THREAD

**Discussion by kymckay** [osmlab/name-suggestion-index#6168] [General]
What is a format?
-

According to the OSM wiki this is one of a few "well established terms used in the British Retail Trade". I've been unable to find it defined elsewhere, but for our purposes the wiki definition is good:
>An example of different formats are Tesco Extra, Tesco Metro and Tesco Express. At the moment we do not tag individual formats.

How should we handle them?
-

At the moment we're not consistent in their handling. Personally I think different store formats should all share the same `brand` tag at least and potentially the same `brand:wikidata` tag too. However, we should come to some consensus either way and stick with it.

Previous Discussion
-
https://github.com/osmlab/name-suggestion-index/pull/2231#discussion_r244643936
https://github.com/osmlab/name-suggestion-index/issues/2740#issuecomment-498042862
https://github.com/osmlab/name-suggestion-index/issues/38#issuecomment-421456527
https://github.com/openstreetmap/iD/issues/6443#issuecomment-497713875

**Comment by Adamant36** [comment]
Where exactly is that quote on the wiki? I didn't see it in the Supermarket article. 

As you see it, what's the harm in not coming to some consensus on it and what would be the benefit of coming to one? Also, why exactly do you think different store formats should all share the same brand/brand:wikidata tags? (I'm only asking to understand the question/problem better).

**Comment by kymckay** [comment]
It's from here: https://wiki.openstreetmap.org/wiki/Retail_chains_in_the_United_Kingdom#Terminology

As for the motivation behind this issue, I've just seen a few people bring it up lately and it's always good to improve our tagging practise if there's a possibility to. I think we should reach a consensus because it sets a precedent for future cases where someone is unsure and means we're all following the same rule which makes the data more homogeneous.

I suppose the reason I think they should share the same tags is because by definition (at least, following this idea of a format) they are the same brand. It also establishes that there is some relation between the features of different formats.

**Comment by bhousel** [comment]
In cases where each store format has its own Wikidata, we link to the sub-brand QIDs. This is great when the businesses use different branding, like in the case with Carrefour:

<img width="1363" alt="Screenshot 2019-06-12 08 21 28" src="https://user-images.githubusercontent.com/38784/59350854-19d36b80-8ceb-11e9-96f9-9f54c5856092.png">

But where it hasn't been split up into sub-brands, we just assign them all the same Wikidata QID:

<img width="1368" alt="Screenshot 2019-06-12 08 23 16" src="https://user-images.githubusercontent.com/38784/59350983-5d2dda00-8ceb-11e9-8909-a0b7642356a9.png">

So, I think _map software can provide a better experience for users_ when store formats are split up and use different `brand:wikidata` where possible.  I think it's also ok to just use the same `brand:wikidata` for different store formats where this level of detail isn't helpful or available yet.

I don't have strong opinions about what we put into the `brand` tag.  It makes most sense to me to have it be a human-readable string that describes what the `brand:wikidata` tag is pointing to.

**Comment by 1ec5** [comment]
By the way, when creating a Wikidata item for a sub-brand, use the reciprocal [parent organization](https://www.wikidata.org/wiki/Property:P749) and [subsidiary](https://www.wikidata.org/wiki/Property:P355) properties to keep the item linked to the broader web of Wikidata items, to keep it from getting deleted as non-notable.

**Comment by kymckay** [comment]
Isn't that not valid for a format though? Because it's not necessarily a separate organisation

**Comment by Adamant36** [comment]
> Isn't that not valid for a format though? Because it's not necessarily a separate organisation

It seems to really depend on what the preferences are of the admin that comes across the article at the time. I've seen plenty of articles that have been there for a few years with no problem that essentially have nothing linked except parent organization. 

There is this quote in the nobility guidelines that makes me think it would be OK as a nobility criteria "It fulfills some structural need, for example: it is needed to make statements made in other items more useful," but that being said, FoodMaxx's article was deleted even with a parent organization/subsidiary reference. I asked @matkoniecz to see if he could get confirmation from an admin if it would qualify when he was messaging them, but unfortunately it never happened and they seem hesitant to give specifics. So, I'm pretty sure the particular admin is what matters most. Along with how many references we use.

In general, we shouldn't hedge our bets on any single reference anyway. The more we add, the less chance there is of the article getting deleted. One type of reference might work for certain kinds of subjects, but not others, who knows. Personally, I see it as a "weighted" thing based on the perceived legitimacy of each source. So I think we should have a list of possibilities, where we throw everything from at the wall that we can. Instead of just picking one reference for everything like the contributing file currently suggests. That's a different topic though.
